warehouse/integrations/redshift/redshift_test.go

```diff
@@ -801,589 +802,587 @@ func TestIntegration(t *testing.T) {
 		require.NoError(t, db.Ping())
 		t.Cleanup(func() { _ = db.Close() })
 
-		testCases := []struct {
-			name      string
-			warehouse model.Warehouse
-		}{
-			{
-				name: "With password",
-				warehouse: model.Warehouse{
-					Source: backendconfig.SourceT{
-						ID: "test_source_id",
-					},
-					Destination: backendconfig.DestinationT{
-						ID: "test_destination_id",
-						Config: map[string]interface{}{
-							"host":             credentials.Host,
-							"port":             credentials.Port,
-							"user":             credentials.UserName,
-							"password":         credentials.Password,
-							"database":         credentials.Database,
-							"bucketName":       credentials.BucketName,
-							"accessKeyID":      credentials.AccessKeyID,
-							"accessKey":        credentials.AccessKey,
-							"namespace":        namespace,
-							"syncFrequency":    "30",
-							"enableSSE":        false,
-							"useRudderStorage": false,
-						},
-						DestinationDefinition: backendconfig.DestinationDefinitionT{
-							ID:          "1UVZiJF7OgLaiIY2Jts8XOQE3M6",
-							Name:        "RS",
-							DisplayName: "Redshift",
-						},
-						Name:        "redshift-demo",
-						Enabled:     true,
-						RevisionID:  "29HgOWobrn0RYZLpaSwPIbN2987",
-						WorkspaceID: "test_workspace_id",
-					},
-					WorkspaceID: "test_workspace_id",
-					Namespace:   namespace,
-				},
+		warehouse := model.Warehouse{
+			Source: backendconfig.SourceT{
+				ID: "test_source_id",
 			},
-			{
-				name: "with IAM Role",
```

Comment:
There is no point in running separate test for password vs Iam role. We already have covered these test above.